src/snark/libsnark/common/profiling.cpp

```diff
@@ -45,10 +45,10 @@ long long get_nsec_cpu_time()
     return ts.tv_sec * 1000000000ll + ts.tv_nsec;
 }
 
```

Comment:
Add a comment saying that these times are nanoseconds since Unix epoch. (Optional, if you are feeling sufficiently nitpicky: "excluding leap seconds".)